parse.y

```diff
@@ -2911,6 +2911,15 @@ rb_parser_ary_free(rb_parser_t *p, rb_parser_ary_t *ary)
 /*
  *	parameterizing rules
  */
+%rule backref_with_rhs(value) <node>
```

Comment:
There are other `backref` errors in `mlhs_node` and `lhs`, and these have also RHS as they are LHS.
This name doesn't look appropriate for op-assign rules only.

Reply:
Certainly.
I'm thinking of renaming it backref_with, but how do you think?

Reply:
It was renamed to `backref_with`.